style.css

```diff
@@ -66,326 +69,413 @@ body {
     top: 0;
     left: 0;
     /* padding:0.2rem; */
-    padding-top:1rem;
+    padding-top: 1rem;
     /* position:sticky; */
-    
+
 }
-.nav-list{
+
+.nav-list {
     list-style: none;
     /* padding-top:1rem; */
     /* padding-left:0.2rem; */
     padding: 1rem 0.2rem;
     /*  */
 }
-.nav-list li{
-    margin:1rem;
-    border-bottom:1px solid var(--background-color);
+
+.nav-list li {
+    margin: 1rem;
+    border-bottom: 1px solid var(--background-color);
     text-align: center;
-    padding:1rem;
+    padding: 1rem;
     line-height: 3rem;
 }
+
 .nav-item {
     text-decoration: none;
     color: var(--background-color);
-    
-    font-size:1.5rem;
+
+    font-size: 1.5rem;
 }
+
 .main {
     grid-area: main;
-    margin:1rem;
+    margin: 1rem;
     padding-top: 1rem;
 }
 
-.main .section{
-    padding:1rem 0.2rem;
+.main .section {
+    padding: 1rem 0.2rem;
     background-color: var(--text-color);
     margin-bottom: 1rem;
 }
-.section-header{
-        font-size: 1.8rem;
-        font-weight: bold;
-        color:var(--background-color);
-        text-align: center;
-        /* background-color: #ffffff; */
+
+.section-header {
+    font-size: 1.8rem;
+    font-weight: bold;
+    color: var(--background-color);
+    text-align: center;
+    /* background-color: #ffffff; */
 }
 
-.container-info{
-    padding:1rem;
-    display:flex;
+.container-info {
+    padding: 1rem;
+    display: flex;
     flex-direction: column;
 
 }
-.info{
+
+.info {
     /* justify-items: center; */
     text-align: justify;
-    margin:0 0.5rem;
+    margin: 0 0.5rem;
 }
-.badge-container{
+
+.badge-container {
     display: flex;
     flex-direction: column;
-    padding:1rem;   
+    padding: 1rem;
     border-bottom: 2px solid var(--off-white);
     /* justify-content: center; */
     /* align-items: center; */
 }
+
 /*---------------- badge----------------------- */
 
-.text-badge{
-    font-size:large;
+.text-badge {
+    font-size: large;
     display: flex;
     justify-content: flex-start;
-    padding:1rem;
+    padding: 1rem;
     border-bottom: 1px solid;
 }
-.badge{
+
+.badge {
     border-radius: 0.5rem;
-    border:2px solid var(--off-white);
-    margin:0.5rem;
-    padding:1rem;
-    display:flex;
+    border: 2px solid var(--off-white);
+    margin: 0.5rem;
+    padding: 1rem;
+    display: flex;
     flex-direction: column;
     align-items: center;
 }
-#badge .fa-home{
+
+#badge .fa-home {
     position: relative;
     font-size: 2.5rem;
 }
-#badge .badge-on-icon{
+
+#badge .badge-on-icon {
     font-size: 1.5rem;
     position: absolute;
     border-radius: 50%;
     background-color: var(--background-color);
-    color:var(--text-color);
-    width:25px;
+    color: var(--text-color);
+    width: 25px;
     text-align: center;
-    left:25px;
-    top:-10px;
+    left: 25px;
+    top: -10px;
```

Comment:
One suggestion try to use one CSS unit everywhere in the project. It Maintains consistency in units.

Reply:
I will implement it. Thank you